### PR TITLE
Module 6

### PR DESCRIPTION
**Pull request by Shibbaz**
## Describe your changes
I've been working on task from module 6.
## Checklist before requesting a review
- [x] I have added pagination and it's working
- [x] I've created rake script
- [x] I've bundled and configured new gems. 
- [x] I've refactored my solution

### CODE REVIEW

**Comment by Shibbaz** [comment]
By accident I closed pr for module-6

File: Gemfile [hunk 1]
```diff
@@ -83,3 +83,6 @@ group :test do
   gem 'selenium-webdriver'
   gem 'webdrivers'
 end
+gem 'simplecov', require: false, group: :test
+gem 'standard', group: %i[development test]
```
Comment (DenisNikolski) [inline Gemfile:87]: just place them in group with others gems

File: app/views/movies/index.html.erb [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
+               <% end %>
+             </tbody>
+            </table>
+            <br>
+            <%= link_to 'New Movie', new_movie_path %>            
+            </ul>
+        </div>
+        
+    </div>
+</nav>
```
Comment (DenisNikolski) [inline app/views/movies/index.html.erb:33]: new line pls

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -1,14 +1,33 @@
...
+               <% end %>
+             </tbody>
+            </table>
+            <br>
+            <%= link_to 'New User', new_user_path %>            
+            </ul>
+        </div>
+        
+    </div>
+</nav>
```
Comment (DenisNikolski) [inline app/views/users/index.html.erb:33]: new line pls

File: spec/requests/movies/get_spec.rb [hunk 4]
```diff
@@ -7,11 +7,11 @@
   describe 'JSON format' do
     describe 'GET /movies' do
       context 'when there are records' do
-        before { create_list(:movie, 10) }
+        before { create_list(:movie, 5) }
```
Comment (DenisNikolski) [inline spec/requests/movies/get_spec.rb:10]: let's leave 10 movies in DB and check that pagination works,  so only 5 records are returned,
then we can check that param page works, so when passing the nexts page, next 5 movies are returns 

same for users

File: Gemfile [hunk 5]
```diff
@@ -62,24 +62,27 @@ group :development, :test do
...
+  gem 'brakeman'
   gem 'web-console'
-
   # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
   # gem "rack-mini-profiler"
 
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
-  gem 'brakeman'
+  gem 'rspec-rails', '~> 6.0.0.rc1'
```
Comment (DenisNikolski) [inline Gemfile:77]: you already have `rspec-rails` in dev and test and it's fine

File: lib/tasks/cleaner.rake [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+namespace :cleaner do
+  desc 'If you want to run rubocop'
+  task rubocop: :environment do
+    system('rubocop -a')
```
Comment (DenisNikolski) [inline lib/tasks/cleaner.rake:6]: you can use RuboCop raketask `RuboCop::RakeTask`

```ruby
require 'rubocop/rake_task'

RuboCop::RakeTask.new(:rubocop) do |t|
  t.options = ['--display-cop-names']
end
```

File: lib/tasks/cleaner.rake [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  end
```
Comment (DenisNikolski) [inline lib/tasks/cleaner.rake:7]: what about other gems like brakeman, simplecov, rubycritic?

File: app/views/users/index.html.erb [hunk 8]
```diff
@@ -1,14 +1,33 @@
...
-  <% @users.each do |user| %>
-    <%= render user %>
-    <p>
-      <%= link_to "Show this user", user %>
-    </p>
-  <% end %>
-</div>
-
-<%= link_to "New user", new_user_path %>
+<nav class="navbar navbar-dark navbar-static-top navbar-expand-md">
```
Comment (DenisNikolski) [inline app/views/users/index.html.erb:1]: i'm not sure if these bootstrap class names will work, 
as I see you don't add any assets/js from bootstrap

Comment (Shibbaz) [reply app/views/users/index.html.erb:1]: Well there was about prettify new view, so that's not exactly bootstrap we wanted, just different solution.

Comment (DenisNikolski) [reply app/views/users/index.html.erb:1]: it's fine, 
but I can't see any CSS styles for these classes

Comment (Shibbaz) [reply app/views/users/index.html.erb:1]: should I leave it as It's now? It's more readable than before and that was kind of the summary of the task (to prettify it). Is bootstrap optional?

Comment (DenisNikolski) [reply app/views/users/index.html.erb:1]: How did you pretify app UI?
I mean, you just added html classes, 
But don't add css,
So I suppose these classes will do nothing 

Regarding bootstrap, it's not required, 
bit it's easiest way to add some styles 

If you don't want use bootstrap,  you can write your own css, or use other css lib

**Review by DenisNikolski — APPROVED** [review]

**Pull request merged**